core/api/service/system/requests/health.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/service/system/requests/health.hpp"
+#include "api/jrpc/value_converter.hpp"
+
+namespace kagome::api::system::request {
+
+  Health::Health(std::shared_ptr<SystemApi> api) : api_(std::move(api)) {
+    BOOST_ASSERT(api_ != nullptr);
+  }
+
+  outcome::result<void> Health::init(
+      const jsonrpc::Request::Parameters &params) {
+    if (!params.empty()) {
+      throw jsonrpc::InvalidParametersFault("Method should not have params");
+    }
+    return outcome::success();
+  }
+
+  outcome::result<jsonrpc::Value::Struct> Health::execute() {
+    jsonrpc::Value::Struct data;
+
+    // isSyncing - Whether the node is syncing.
+    data["isSyncing"] = makeValue(api_->getBabe()->getCurrentState()
+                                  == consensus::Babe::State::SYNCHRONIZED);
+
+    // peers - Number of connected peers
+    data["peers"] = makeValue(api_->getGossiper()->getActiveStreamNumber());
+
+    // shouldHavePeers - Should this node have any peers.
+    // Might be false for local chains or when running without discovery.
+    data["shouldHavePeers"] =
+        makeValue(api_->getGossiper()->getReservedStreamNumber() != 0);
```

Comment:
After executing this RPC agains Substrate I noted that it returns false when ChainType is "Development" and true otherwise. Maybe do the same in our implementation?

Reply:
Why not? Done.